daffodil-test/src/test/resources/edu/illinois/ncsa/daffodil/section07/assertions/assert.tdml

```diff
@@ -639,11 +639,12 @@
 
   <tdml:parserTestCase name="assertOnGroupRef" root="Item"
     model="s1" description="assert on a group reference - DFDL-7-050R">
-
     <tdml:document><![CDATA[Shirts,30]]></tdml:document>
     <tdml:errors>
-      <tdml:error>Assert</tdml:error>
-      <tdml:error>pattern</tdml:error>
+      <tdml:error>Parse Error</tdml:error>
+      <tdml:error>Assertion</tdml:error>
```

Comment:
Just say "Assert" , as we really don't care if the word "assertion" is used in the message vs say, dfdl:assert.